Makefile

```diff
@@ -118,8 +118,8 @@ endif
 
 bin/z88dk-zsdcc$(EXESUFFIX): $(SDCC_PATH)/bin/sdcc$(EXESUFFIX)
 ifdef BUILD_SDCC
-	cp $(SDCC_PATH)/bin/sdcc$(EXESUFFIX)  $(Z88DK_PATH)/bin/z88dk-zsdcc$(EXESUFFIX)
-	cp $(SDCC_PATH)/bin/sdcpp$(EXESUFFIX) $(Z88DK_PATH)/bin/z88dk-zsdcpp$(EXESUFFIX)
+	cp $(SDCC_PATH)/src/sdcc$(EXESUFFIX)  $(Z88DK_PATH)/bin/z88dk-zsdcc$(EXESUFFIX)
```

Comment:
?? Seems a bit odd

Reply:
The executables are no longer copied to `sdcc/bin` on creation. There are just scripts from `sdcc/bin` that call them in place.

So to find the actual executables you have to dig a bit deeper.

Had me guessing for a few minutes, why the `sdcc` was suddenly only 200 odd Bytes.

You don't normally "open" a bin file to check the contents, so I thought the build had gone bad on first pass.

Reply:
See https://github.com/z88dk/z88dk/issues/2186#issuecomment-1424784546